src/scss/header.scss

```diff
@@ -17,6 +17,15 @@
   app-search {
     grid-column-start: 2;
     width: 100%;
+
+    [type="search"]::-webkit-search-cancel-button {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 15px;
+      width: 15px;
+      background-repeat: no-repeat;
+      background-image: url("../images/close-white.png");
```

Comment:
We can't use a constant image fo this, it needs to be themified since the nord theme uses a white background to the search bar :(
<img width="629" alt="image" src="https://user-images.githubusercontent.com/18214891/165948613-b5aa7c84-4ca2-4c54-b1d0-2384f8761c06.png">

Reply:
Thanks, nice catch! I'll update that shortly 😄

Reply:
One possible approach is to use an svg image and use the css fill attribute to match whichever theme is active. This will avoid needing to use multiple files.

Reply:
What is your thoughts on using a mask-image instead? It would allow us to re-use the text color.

```
        mask-image: url("../images/close-button.svg");
        -webkit-mask-image: url("../images/close-button.svg");
        background-color: #fff;
```